docker-compose.yaml

```diff
@@ -64,17 +72,16 @@ services:
     #       deployments.
     extra_hosts:
       - "localhost:host-gateway"
+    profiles: 
+      - !external
```

Comment:
From the documentation it looks like we'll be relying on an external S3-compatible object store too. Should we also disable localstack with `!external`?

Reply:
It definitely makes sense to add !external to this. The only caveat is that I would need to remove the depends_on in the backend section and I'm not entirely happy with it. I've tested it and it works, but it might be because I have things cached locally, if I do a fresh start it might fail without the depends_on. If the users want to launch lumigator with external ray and I don't add the !profile as you suggest, the only issue right now is that the user will have an extra container locally. I think a possible solution it's to divide the compose into different files and load them on demand with the Makefile. Would you be happy to be addressed as a refactor in a different ticket?

Reply:
Thanks for explaining! I think it's ok, let us just track this via an issue and merge the current code